src/gui/vector/qgsfieldcalculator.cpp

```diff
@@ -481,7 +486,7 @@ void QgsFieldCalculator::mOutputFieldTypeComboBox_activated( int index )
   setPrecisionMinMax();
 }
 
-void QgsFieldCalculator::populateFields()
+void QgsFieldCalculator::populateFields( const int &fieldIndex )
```

Comment:
```suggestion
void QgsFieldCalculator::populateFields( int fieldIndex )
```

There's no need to pass small types like int by reference -- it's actually more work to do this vs a copy.